include/ensmallen_bits/problems/ackley_function.hpp

```diff
@@ -112,10 +112,10 @@ class AckleyFunction
   //! Modify the value used for numerical stability.
   double& Epsilon() { return epsilon; }
 
-  // Note: GetInitialPoint(), GetFinalPoint(), and GetFinalObjective() are not
-  // required for using ensmallen to optimize this function!  They are
-  // specifically used as a convenience just for ensmallen's testing
-  // infrastructure.
+  // Note: GetInitialPoint(), GetFinalPoint(), and GetFinalObjective(),
+  // GetVariablesBounds() are not required for using ensmallen to
```

Comment:
```suggestion
  // Note: GetInitialPoint(), GetFinalPoint(), GetFinalObjective(), and
  // GetVariablesBounds() are not required for using ensmallen to
```
Just a quick grammar fix. :)